packages/suite/src/components/suite/modals/PassphraseDuplicate/index.tsx

```diff
@@ -22,6 +22,14 @@ const StyledImage = styled(Image)`
     margin: 24px 0px;
 `;
 
+const StyledModal = styled(Modal)`
+    width: 360px;
```

Comment:
In this case it will still be easier to change the Modal if width is handled as a prop, no need to remove / edit styled components. Up to you, feel free to resolve :)